### PR TITLE
docs: update README file

### PR DESCRIPTION
**Pull request by shmily7829**
#1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,3 +25,28 @@
...
+
+## GitHub Flow
+
+1. 團隊任一人皆可依據功能建立issue，並依據需求類型添加lebel及指派人員處理
+2. 分支的建立規則：
+    1. 分支名稱為dev和左斜線與 issue 編號結合
+    2. Pattern: `dev/${issueId}`，範例： feat/1 
+3. 本地開發測試完成之後，並push本地分支到遠端
+4. 發出Pull Request，於commit裡填寫issue link，請求Reviewer Review Code
+5. 需有2人Review PR
```
Comment (c931534) [inline README.md:48]: 前後端大家都可以review，這邊可以改一下說明喔

File: README.md [hunk 2]
```diff
@@ -36,3 +25,28 @@
...
+- **chore:** 建構程序或輔助工具的變動 (maintain)。
+- **revert:** 撤銷回覆先前的 commit 例如：revert: type(scope): subject (回覆版本：xxxx)。
+
+
+## GitHub Flow
+
+1. 團隊任一人皆可依據功能建立issue，並依據需求類型添加lebel及指派人員處理
+2. 分支的建立規則：
+    1. 分支名稱為dev和左斜線與 issue 編號結合
+    2. Pattern: `dev/${issueId}`，範例： feat/1
```
Comment (IamSaker) [inline README.md:45]: 範例：dev/1

File: README.md [hunk 3]
```diff
@@ -36,3 +25,28 @@
...
+
+1. 團隊任一人皆可依據功能建立issue，並依據需求類型添加lebel及指派人員處理
+2. 分支的建立規則：
+    1. 分支名稱為dev和左斜線與 issue 編號結合
+    2. Pattern: `dev/${issueId}`，範例： feat/1 
+3. 本地開發測試完成之後，並push本地分支到遠端
+4. 發出Pull Request，於commit裡填寫issue link，請求Reviewer Review Code
+5. 需有2人Review PR
+    1. 後端Reviewer：Saker、Fuyao
+    2. 前端Reviewer：待定
```
Comment (IamSaker) [inline README.md:50]: Mahjong Team members could review the PR.

File: README.md (hunk 2 above)
Comment (c931534) [inline README.md:45]: 看不太懂這邊的說明，dev/feat/1 ?

**Review by c931534 — COMMENTED** [review]
review done. please check.

**Comment by IamSaker** [comment]
Nitpick: 我建議參考 [中文文案排版指北](https://github.com/sparanoid/chinese-copywriting-guidelines)
README 或文件的排版，建議都參考這篇文。

File: README.md (hunk 2 above)
Comment (shmily7829) [reply README.md:45]: dev/1

**Comment by shmily7829** [comment]
Update README.md , @IamSaker  @c931534 Please Check.

**Review by IamSaker — APPROVED** [review]

**Review by c931534 — APPROVED** [review]

**Pull request merged**